data_api/snapshot_convertor/src/test/scala/uk/ac/wellcome/platform/snapshot_convertor/flow/StringToGzipFlowTest.scala

```diff
@@ -0,0 +1,72 @@
+package uk.ac.wellcome.platform.snapshot_convertor.flow
+
+import java.io.File
+import java.nio.file.Paths
+
+import akka.actor.ActorSystem
+import akka.stream.scaladsl.{FileIO, Flow, Keep, Sink, Source}
+import akka.stream.{ActorMaterializer, IOResult, Materializer}
+import akka.util.ByteString
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.{FunSpec, Matchers}
+import scala.io.Source.fromFile
+import uk.ac.wellcome.test.fixtures.Akka
+
+import scala.sys.process._
+import scala.concurrent.{ExecutionContextExecutor, Future}
+
+class StringToGzipFlowTest
+    extends FunSpec
+    with Matchers
+    with Akka
+    with ScalaFutures {
+
+  it("produces a gzip-compressed file from the lines") {
+    withActorSystem { actorSystem =>
+      implicit val executionContext: ExecutionContextExecutor =
+        actorSystem.dispatcher
+      implicit val system: ActorSystem = actorSystem
+      implicit val materializer: Materializer =
+        ActorMaterializer()(actorSystem)
```

Comment:
I think I saw a `withActorMaterialiser` fixture while reviewing another PR (can't remember which PR...)

Reply:
Yep, that’s in at least two other PRs. I’ll retrofit it to #1748. 😛